numba/tests/test_target_extension.py

```diff
@@ -317,6 +323,8 @@ def dispatcher_wrapper(self):
         if "nopython" in self._kwargs:
             topt["nopython"] = True
 
+        topt['target_backend'] = 'dpu'
```

Comment:
This is going to be removed by #9539.

Reply:
Yes, I am unconvinced it had any effect.

Reply:
Thanks - I've now removed it.